clammit.cfg.example

```diff
@@ -31,3 +31,9 @@ log-file        = log/clammit.log
 # the virus scanning
 #
 #test-pages      = true
+
+#
+# If true, will skip SSL validation forwarding connection to use self-signed certitifates
+# default = false
```

Comment:
Also here I would reword the description as:

> if true, it will not perform TLS certificate verification, allowing connections to servers using self-signed or otherwise untrusted certificates. Use with care, it is always preferable to perform certificate verification.

Reply:
I like your very precise documentation